src/garage/experiment/local_tf_runner.py

```diff
@@ -241,8 +248,11 @@ def restore(self, snapshot_dir, from_epoch='last'):
             specify manually or through run_experiment() interface.
 
         """
-        snapshotter.snapshot_dir = snapshot_dir
-        saved = snapshotter.load(from_epoch)
+        assert isinstance(self.snapshot_config, SnapshotConfig), (
+            'You must initialize SnapshotConfig in LocalRunner before '
```

Comment:
This does not give instruction on where to load the snapshot. What about 'You have to specify saved snapshot folder in construction via SnapshotConfig'.